src/Recipe.js

```diff
@@ -12,9 +16,22 @@ class Recipe {
     return this.instructions;
   }
 
-  calculateTotalRecipeCost() {
-    // return the total cost = amount * price
-    //need an ingredient class
+  getIngredientWithPrices() {
+    let ingredientsWithPrice = prototypeIngredients.filter(ingredient => {
+     return this.ingredients.find(recipeIngredient => {
+       return   recipeIngredient.id === ingredient.id;
+     });
+    });
+    return ingredientsWithPrice;
+  }
+
+  calculateTotalCost() {
```

Comment:
Nice! I see what you were doing here. This makes sense for what we are trying to accomplish!